app/operations/event_procedures/build_total_amount_cents.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module EventProcedures
+  class BuildTotalAmountCents < Actor
+    input :event_procedure, type: EventProcedure
+
+    output :total_amount_cents, type: Integer
+
+    APARTMENT_PERCENTAGE = 1
+    URGENCY_PERCENTAGE = 0.3
+
+    def call
+      self.total_amount_cents = event_procedure.procedure.amount_cents + apartment_amount + urgency_amount.to_i
```

Comment:
why `urgency_amount.to_i` and `apartment_amount` does not call `to_i` ?

Reply:
because multiplication in case of urgency, would return a broken Float type number.

Ex:
the value 10100 (which is equivalent to R$101.00) * 0.3 (30%) would return the value 3030.0.
then we converted this value to Integer, and in the end we would have 3030 (of the Integer type) which would be equal to R$30.30